spec/doublyLinkedList.js

```diff
@@ -0,0 +1,226 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import DoublyLinkedList from '../src/doublyLinkedList'
+
+chai.use(chaiChange)
+
+describe('DoublyLinkedList', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(DoublyLinkedList).to.be.a('function')
+  })
+
+  context('insert()', () => {
+    it('adds a new node to the linked list when empty', () => {
+      const myDoublyLinkedList = new DoublyLinkedList()
+      myDoublyLinkedList.insert("Azazel")
+      expect(myDoublyLinkedList.size()).to.eql(1)
+    })
+    it('adds new node to the end if list is not empty', () => {
+      const myDoublyLinkedList = new DoublyLinkedList()
+      myDoublyLinkedList.insert("Azazel")
+      myDoublyLinkedList.insert("Beelzebub")
+      myDoublyLinkedList.insert("Choronzon")
+      expect(myDoublyLinkedList.size()).to.eql(3)
+      expect(myDoublyLinkedList.getHeadNode().next.next.previous.data).to.eql("Beelzebub")
```

Comment:
Why are there so many calls strung together? I think you could have separated this out into two separate expect statements. One using getHeadNode().next.data and one using getTailNode().previous.data. Also, I expected the next and previous members to be accessed by a function that returns that value, not directly accessed as they are here.